code/modules/paperwork/photocopier.dm

```diff
@@ -184,6 +184,15 @@
 			usr << "<span class='notice'>You take [photocopy] out of [src].</span>"
 			photocopy = null
 			updateUsrDialog()
+		else if(doccopy)
+			if(!istype(usr,/mob/living/silicon/ai)) // honk honk honk

```

Comment:
Don't put silly comments like "// honk honk honk", if you add a comment, at least make it one that explains what the code does.